transformer/dockerfilegenerator/dotnetcoredockerfilegenerator.go

```diff
@@ -38,8 +37,11 @@ import (
 )
 
 var (
-	portRegex            = regexp.MustCompile(`:(\d+)`)
-	dotNetCoreFrameworks = []string{"net5.0", "netcoreapp2.1", "netstandard2.0"}
+	portRegex        = regexp.MustCompile(`:(\d+)`)
+	versionRegex     = regexp.MustCompile(`\d+\.\d+`)
+	dotnetcoreRegex1 = regexp.MustCompile(`net\d+\.\d+`)
+	dotnetcoreRegex2 = regexp.MustCompile(`netcoreapp\d+\.\d+`)
+	dotnetcoreRegex3 = regexp.MustCompile(`netstandard\d+\.\d+`)
```

Comment:
Do we need 4 regex here? Can we use something like `net(?:(?:coreapp)|(?:standard))?(\d+\.\d+)` , and use the capturing group to get the version with just a single regex?

Reply:
Yes sure, thank you. I have updated the regex.